src/types/earn_program/list_validator.rs

```diff
@@ -0,0 +1,5 @@
+use cosmwasm_schema::cw_serde;
+
+#[cw_serde]
+pub struct ListValidators {
```

Comment:
@kenta-elys  is it normal that the `ListValidators` struct is empty ?

Reply:
@politeWall Yes it is empty but is created for dummy response implementation.